pkg/webhook/image_swapper.go

```diff
@@ -70,8 +72,9 @@ func Copier(pool *pond.WorkerPool) Option {
 
 // ImageSwapper is a mutator that will download images and change the image name.
 type ImageSwapper struct {
-	registryClient          registry.Client
-	imagePullSecretProvider secrets.ImagePullSecretsProvider
+	sourceRegistryClients     []registry.Client
+	destinationRegistryClient registry.Client
```

Comment:
Can we rename this to `targetRegistryClient` to stay consistent with the config?